### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by richardolsson** [zetkin/app.zetkin.org#254]
We should make a proper accessibility audit of the entire Gen 3 interface.

- [ ] Registration and login flow
- [ ] Public pages
- [ ] Organizer pages

**Comment by djbusstop** [comment]
Apparently [Axe](https://www.deque.com/axe/) is a good accessibility auditer. It has firefox and chrome plugins.

**Comment by jones58** [comment]
I suggest a wider accessibility audit will be needed as we work through implementing the new design system (#2111) and moving to the app router (can't find an issue for this). For now i'm focussing on issues in the app folder.

**Comment by jones58** [comment]
# Current Issues

- Many elements on the site are lacking sufficient colour contrast. In particular the light grey colour used and bright red do not meet the [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/).
- Many buttons do not feature text or tooltips. 
- Images lack alt text. SVGs should have a title property.
- Site is not navigable at 400%. Navigation breaks up at 240% (NB this has been fixed on the App Router)
- Many pages do not feature a title, or where they do, they do not differentiate sufficiently. 

# How We Can Improve

### Buttons

- Buttons without text should feature tooltips where possible.

### Automated Testing

- There are several tools available for basic accessibility testing:
  - [Axe Developer Tools Extension](https://www.deque.com/axe/)
  - [Unlighthouse](https://next.unlighthouse.dev/) on the command line.
  - [WAVE Browser Extension](https://wave.webaim.org/)

### Manual Testing

- Run through this [checklist](https://www.a11yproject.com/checklist/).
- Use the WCAG Color contrast checker extension to check and change colours across the site and ZUI components as they are built.

### User Testing
- 
- Test with users with access needs (low vision,autism, dyslexia, colour blindness, physical disabilities).

### Developing an accesibility statement

- This can be as simple as this [site](https://www.coop.co.uk/accessibility) or more complex such as [this](https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website). Developers and designers should be made aware of this before working on the site.


### More resources
- [Accessibility Developer Guide](https://www.accessibility-developer-guide.com/)
- [Folder of Accessibility Tools](https://www.are.na/gemma-copeland/accessibility-xkrrbesezrm)
